composer.json

```diff
@@ -13,8 +13,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^5.3 | ^7.0",
-        "dhii/data-identifiable-interface": "^0.1.0",
-        "dhii/validation-interface": "^0.1.0"
```

Comment:
Why is validation interface package not a requirement?

Reply:
Nothing in this package needs validation interfaces.